csharp-basic-topics/ValueVsReferenceTypes/ValueVsReferenceTypes/Car.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ValueVsReferenceTypes
+{
+    public class Car
+    {
+        public string ChangeCarModel( string newModel)
```

Comment:
Remove the blank space before the `string` keyword in the method parameters space.